README.md

```diff
@@ -11,7 +11,7 @@ Authors
 - Created by [Terry Cavanagh](http://distractionware.com/)
 - Room Names by [Bennett Foddy](http://www.foddy.net)
 - Music by [Magnus Pålsson](http://souleye.madtracker.net/)
-- Metal Soundtrack by [FamilyJules](http://familyjules7x.com/)
+- Metal Soundtrack by [FamilyJules](https://link.space/@familyjules)

```

Comment:
Huh, that's weird. It definitely works for me. You could apply this change, tho, and commit?

```suggestion
- Metal Soundtrack by [FamilyJules](https://youtube.com/@FamilyJules7X)
```

Reply:
Yep, that works! Thanks!